src/home/home.scss

```diff
@@ -43,61 +42,34 @@
         white-space: normal;
       }
 
-      .buttons {
-        display: grid;
-        grid-auto-flow: column;
-        justify-content: space-between;
-        cursor: pointer;
-        width: 710px;
-        grid-gap: 24px;
-        .buttonContainer {
-          display: flex;
-          border-radius: 6px;
-          overflow: hidden;
-          & .header {
-            -webkit-text-fill-color: white;
+      .cards {
+        display: flex;
+        pcard {
+          width: clamp(100px, 100%, 360px);
+          .pcard {
+            padding: 0 18px 24px 24px;
           }
-        }
-        .contribute,
-        .launch {
-          border-left-width: 6px;
-          border-right-width: 0;
-          border-left-style: solid;
-          border-image: linear-gradient(to bottom, #ff497a, #a258a7) 1 100%;
-          border-radius: 6px;
-          background-color: $BG02;
-          width: 300px;
-
-          padding: 18px;
-
-          .title {
+          header {
             display: flex;
+            font-size: 28px;
+            line-height: 65px;
             justify-content: space-between;
+            align-items: center;
             color: $White;
-            margin-bottom: 16px;
-            margin-right: 6px;
-
-            .header {
-              font-weight: bold;
-              font-family: "Aeonik";
-              font-size: 28px;
-              white-space: nowrap;
-              padding-right: 16px;
-            }
-
-            .arrow {
-              font-size: 28px;
+            .icon {
+              font-weight: 400;
             }
           }
-
           .subtitle {
-            color: $Neutral03;
-            font-size: 16px;
             line-height: 20px;
-            font-weight: 400;
-            font-family: "Inter";
+            color: $Neutral03;
           }
         }
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: start;
+        cursor: pointer;
```

Comment:
Pointer intended for `.card`?

Nit: I'm suggesting to put css definitions directly under selector (here, it is at the bottom, after all child styles, thus hard to see where it belongs to)

Reply:
agreed about putting at the top